docs/deploy/create_server.rst

```diff
@@ -104,6 +104,60 @@ Hetzner
 
 #. Wait to be notified via email that the server is ready.
 
+Azure
+~~~~~~
+
+#. `Log into Azure <https://login.microsoftonline.com>`__
+#. Click *Virtual machines*
+#. Click *Create* then select *Azure virtual machine*
+
+
+   #. Set *Subscription* to *Microsoft Azure Sponsorship*
+   #. Set *Resource Group* to the appropriate group (e.g. kingfisher).
+      #. Create a new "Resource Group" if required
+   #. Set *Virtual Machine Name* to the server's FQDN (e.g. ``ocp25.open-contracting.org``)
+   #. Set *Region* to *UK South*
+   #. Set *Image* to the latest Ubuntu LTS version
+   #. Set *Size* to an appropriate size. Standard 'starting' size is B2s
+   #. Select *Authentication type* to *Password*
+   #. Set *Username* to ocpadmin
+   #. Set *Root Password* to a `strong password <https://www.lastpass.com/features/password-generator>`__
+
+   #. Select a *Next: Disks >*
+   #. Set *OS Disk Type* to *Standard SSD* 
+      #. Create and attach an additional disk if required with the *Size* of *Standard SSD*
+
+   #. Select a *Next: Networking >*
+   #. Set *Virtual Network* to *SensibleName-vnet (e.g. kingfisher-vnet)*   
+   #. Set *Subnet* to use an IP range that does not overlap an already existing *Virtual Machine* in the same *Resource Group*    
```

Comment:
In what scenario would two virtual machines be in the same resource group?

How is this done?

Reply:
I'll removed this. 
If you had a separate DB/Web server situation, you could place everything around the servers into the same resource group.

Reply:
Bytemark also offered an option to group VMs, but it just became unnecessary overhead. Is there any consequence to just putting everything in a "default" group?